test/integration/lib/expression.js

```diff
@@ -70,19 +70,35 @@ exports.run = function (implementation, options, runExpressionTest) {
 
             if (process.env.UPDATE) {
                 fixture.expected = result;
+                // We save the expected serialization, but the round-tripped compilation results
+                // and outputs should be identical, so no need to save them.
+                delete fixture.expected.recompiled;
+                delete fixture.expected.roundTripOutputs;
```

Comment:
These properties get added by the native test runner but not the JS one, which might make this a bit confusing to maintain later... instead of copying `result` wholesale, let's opt-in to the specific properties we want:
```js
fixture.expected = {
    compiled: result.compiled,
    outputs: result.outputs,
    serialized: result.serialized !== undefined ? result.serialized :
        (fixture.expected && fixture.expected.serialized) ? fixture.expected.serialized :
        fixture.expression
}
```

Reply:
:+1: